src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js

```diff
@@ -30,14 +30,14 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
         };
 
         function isEligibleToEdit() {
-          return $scope.relativeDayNumber < 0 && !$scope.isRecruiter;
+            return $scope.relativeDayNumber < 0 && !$scope.isRecruiter;
         }
 
         function updateDate() {
             startDate = $filter('date')(getDayOfTheWeek(new Date(), $scope.relativeDayNumber), "dd-MM-yyyy");
-            endDate  = $filter('date')(getDayOfTheWeek(new Date(), $scope.relativeDayNumber + 4), "dd-MM-yyyy");
-            $scope.displayedStartDate = startDate.replace(/-/g,'.');
-            $scope.displayedEndDate = endDate.replace(/-/g,'.');
+            endDate = $filter('date')(getDayOfTheWeek(new Date(), $scope.relativeDayNumber + 4), "dd-MM-yyyy");
```

Comment:
The date format string should be refactored into a constant as well. Please check other files for the same string as well.